src/python/WMCore/MicroService/Unified/MSManager.py

```diff
@@ -231,6 +231,8 @@ def info(self, reqName=None):
                 transferDoc = self.msMonitor.reqmgrAux.getTransferInfo(reqName)
             elif 'transferor' in self.services:
                 transferDoc = self.msTransferor.reqmgrAux.getTransferInfo(reqName)
```

Comment:
I would like to sanitize the request name here, by calling:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Lexicon.py#L500
but we can deal with that in a different issue if you prefer.

Reply:
Done!

Reply:
Actually, @amaltaro, this raises a question though. Should we return a `None` transferDoc in case of a bad requestName expression (maybe this could be beneficial if the API is used by another automated interface but not only by a User) or should we return the error so that to inform the User about his mistake?

Reply:
Todor, looking a bit more into this, I think what would make more sense is to validate the GET parameters at the parent class (Service/Data) and return an error like:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/ReqMgr/Service/RequestAdditionalInfo.py#L35
in case of bad request (request name not matching the lexicon).

That work goes orthogonal to these changes though, so I think you can remove the current request name validation here and we deal with that in a different issue/PR. What do you think?

Reply:
OK , Alan@amaltaro. I dropped the reqName validation from `getTransferInfo`